server/graphql_schemas/tests/attend/base.py

```diff
@@ -60,7 +60,7 @@ def setUp(self):
         self.request = RequestFactory().get('/graphql')
         self.client = Client(schema)
 
-    def tearDown(self):
-        Category.objects.all().delete()
-        AndelaUserProfile.objects.all().delete()
-        Event.objects.all().delete()
+    # def tearDown(self):
```

Comment:
You can also skip this in the meantime using this: https://docs.djangoproject.com/en/2.2/topics/testing/tools/#skipping-tests

Reply:
https://docs.djangoproject.com/en/2.2/topics/testing/tools/#skipping-tests

Reply:
Thanks, @andela-sjames for this has been helpfull